com.roundabout_host.Izvor.yaml

```diff
@@ -0,0 +1,70 @@
+app-id: com.roundabout_host.Izvor
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: izvor
+finish-args:
+  - --share=network
+  - --filesystem=home
+  - --filesystem=host
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=wayland
+  - --filesystem=xdg-data/izvor:create
+  - --filesystem=xdg-config/izvor:create
+environment:
+  PYTHONPATH: "/app/lib/python3.12/site-packages:${PYTHONPATH}"
+modules:
+  - name: python-requirements
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/python_packages
+      - cp -r . /app/share/python_packages/
+    sources:
+      - type: dir
+        path: python_packages
+  - name: izvor
+    buildsystem: simple
+    build-commands:
+      - unset GTK_MODULES
+      - mkdir -p /app/bin /app/share/izvor
+      - cp -r ./* /app/share/izvor/
+      - ls /app/share/python_packages
+      - pip install --no-index --find-links=/app/share/python_packages -r /app/share/izvor/requirements-flatpak.txt --target=/app/lib/python3.12/site-packages
+      - chmod +x /app/share/izvor/__init__.py
+      - ln -s /app/share/izvor/__init__.py /app/bin/izvor
+      #- fc-cache -fv
+    sources:
+      - type: dir
+        path: src
+  - name: izvor_utils
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib/python3.12/site-packages/izvor_utils
+      - cp -r . /app/lib/python3.12/site-packages/izvor_utils/
+    sources:
+      - type: dir
+        path: izvor_utils
+  - name: icon
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/icons/hicolor/scalable/apps
+      - cp com.roundabout_host.Izvor.svg /app/share/icons/hicolor/scalable/apps/
+    sources:
+      - type: file
+        path: com.roundabout_host.Izvor.svg
+  - name: metainfo
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 com.roundabout_host.Izvor.metainfo.xml /app/share/metainfo/com.roundabout_host.Izvor.metainfo.xml
+    sources:
+      - type: file
+        path: com.roundabout_host.Izvor.metainfo.xml
+  - name: desktop
+    buildsystem: simple
+    build-commands:
```

Comment:
this 3 "module" should be merged with the main one.

See also the note about what should be upstream.